src/utils/strings/index.ts

```diff
@@ -60,6 +60,18 @@ export const humanReadableFileSize = (size: number) => {
 export const capitalize = (word: string): string =>
   word && word.charAt(0).toUpperCase() + word.slice(1);
 
+/**
+ * Capitalize a word (converts the word to lower case, except for the first letter)
+ *
+ * @method convertToCapitalized
+ *
+ * @param {string} word The word / string to capitalize
+ * @return {string} The capitalized string
+ */
+export const convertToCapitalized = (word: string): string => {
```

Comment:
Different from the function above in that it will first convert the word to lowercase before capitalising the first letter.

Reply:
Yeah, but the naming (for both) is a bit confusing. 

How about `capitalizeFirstLetter` and `capitalizeWord` ?